### DISCUSSION TITLE
pc浏览器进不去luci界面，手机浏览器可以进

### DISCUSSION THREAD

**Discussion by 13528943945** [immortalwrt/immortalwrt#524] [Q&A] [answered]
最近多次用master分支编译，完成后出现pc浏览器进不去luci界面，手机浏览器进luci界面一切正常

**Comment by nicholas-opensource** [comment, marked as answer]
尝试使用隐身模式或者换一个PC浏览器进入，js导致的缓存

**Comment by 13528943945** [comment]
> 尝试使用隐身模式或者换一个PC浏览器进入，js导致的缓存

不是缓存的原因，试了谷歌和火狐，而且先清缓存再打开，依然进不了luci界面，只出现luci-lua configuration interface字样

**Comment by GoingTime** [comment]
分别试试 HTTP 和 HTTPS，有可能是玄学兼容问题

**Comment by ayancn** [comment]
https://github.com/immortalwrt/immortalwrt/discussions/710  

同样的问题    master分支和1806K4都有这个问题

**Comment by xyskwjy** [comment]
也有这问题，ping管理地址通畅，winscp也能进入，就是管理页进不云。隐身模式，清空缓存等等都不行。

**Comment by ayancn** [comment]
没有解决办法吗?

**Comment by ayancn** [comment]
1806k5.4分支也是同样问题

**Comment by nanchuci** [comment]
pc没获取到地址？

**Reply by ayancn** [reply]
> pc没获取到地址？

ip显示6x.x.x.x   最新的coolsnowwolf的版本也是一样问题会好一点就是重启后出现,断电重启才能进,  immortalwrt 直接编译好了电脑就不能进 wifi手机可以进 目前就是官方版本 和 Lienol 版本正常 , 希望能修复  谢谢
路由器是CR6608 (CR660X)

**Comment by ayancn** [comment]
ip显示6x.x.x.x LEAN R2.2.5.5之后最新多个版本都有这个问题重启后出现,断电重启才能进,
immortalwrt master 和 18.06K5.4版本也有这个问题是直接编译好了电脑就不能进(显示拒绝连接)
然后都是wifi手机可以进
目前就是官方版本 和 Lienol 版本正常 , 希望能修复 谢谢
路由器是CR6608 (CR660X)
